src/parser.js

```diff
@@ -69,6 +78,9 @@ const parser = yargs
     //   describe: 'Path to the prettier config to use',
```

Comment:
Could you remove the TODO referenced here for `prettierOptions`. I don't think we'd ever do this with the support you've just added :+1: